nixie/unittests/test_get_renderer_cmd.py

```
@@ -0,0 +1,253 @@
+        assert resolved.backend == "mmdc"
+        assert resolved.needs_puppeteer_config is True
+
+
+class TestGetMermanCmd:
+    """Command construction for the merman backend."""
+
+    def test_command_shape(
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        sample_paths: tuple[Path, Path, Path],
+        fake_home_cwd: Path,
```

Ruff (0.15.15)

[warning] Unused method argument: `fake_home_cwd`

(ARG002)